version-stable-r-development.Rmd

```diff
@@ -83,38 +91,42 @@ following setup:
 - No authentication required (local setup).
 - Enable root by setting the environment variable `ROOT` to `TRUE`, so that e.g.
 `sudo apt-get` can be used in RStudio.
-- Use a version-specific port, e.g. `3500` for R 3.5.0, `3610` for R 3.6.1 and
-so on, so that we can use `localhost` for concurrent R version instances.
+- Use a version-specific port, e.g. `4000` for R 4.0.0, `4410` for R 4.4.1 and
+so on, so that we can use `localhost` for concurrent R version instances. 
+We bind the port to localhost (`127.0.0.1:4410`), so it is only accessible locally 
+(see [the Rocker reference](https://rocker-project.org/images/versioned/rstudio.html#disable_auth)).
 - The development code of all relevant projects should live outside the
-container and be shared with it (and possibly many of them), e.g. under
-`~/RStudioProjects` on the host machine and `/home/rstudio/RStudioProjects` in
+container and be shared with it (and possibly multiple other containers), e.g. under
+`~/workspace` on the host machine and `/home/rstudio/workspace` in
 the container.
     - For this to work w/o [permission
     issues](https://github.com/rocker-org/rocker/wiki/Sharing-files-with-host-machine#avoiding-permission-changes-when-sharing-volumes),
-    the container user (`rstudio`) must match the UID of the host user (`$UID`).
-- In order for the RStudio setting to persist if the container is recreated
+    the container user (`rstudio`) must match the UID of the host user (`$UID`). 
+    This has the effect of setting the ownership of `~/workspace` on the host machine to `$UID` if it is not already owned by that user.
+- In order for the RStudio settings to persist if the container is recreated
 (e.g. after pulling a new `rocker` image), we also use a shared volume (like
-`~/.rstudio-docker/3.6.1`) for the `home/rstudio/.rstudio` directory, which is
-version-specific in case of multiple R versions
+`~/.rstudio-config/4.4.1`) for the `/home/rstudio/.config/rstudio` directory, which is
+version-specific in case of multiple R versions.
 - If we want to use Meld via the [compareWith](https://github.com/miraisolutions/compareWith/) addins, we need to
-    - map the `DISPLAY` environment variable and volume `/tmp/.X11-unix`
-    - add `DISPLAY` to `Renviron`
-    - install Meld
-    - install `dbus-x11`
+    - map the `DISPLAY` environment variable and volume `/tmp/.X11-unix`,
+    - add `DISPLAY` to `Renviron`,
+    - install Meld,
+    - install `dbus-x11`.
 - Use a version-specific name for the container running the RStudio instance,
-e.g. `rstudio_3.6.1`.
+e.g. `rstudio_4.4.1`.
 
 
 ```{bash run}
-R_VER=3.6.1
-SHARED_DIR=RStudioProjects
+R_VER=4.4.1
+SHARED_DIR=workspace
+mkdir -p $HOME/.rstudio-config/$R_VER
 docker run -d --restart=always \
   -p 127.0.0.1:$(echo $R_VER | sed 's/[.]//g')0:8787 \
   -e DISABLE_AUTH=true \
   -e ROOT=TRUE \
```

Comment:
Let's take the chance to align the usage of lowercase `true` as in https://rocker-project.org/images/versioned/rstudio.html#root

```suggestion
  -e ROOT=true \
```